### PR TITLE
Feature login button

### PR DESCRIPTION
**Pull request by ArturSkuratovic**

### CODE REVIEW

File: src/app/navigationbar/navigation-bar.component.html [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
       </button>
     </a>
-    <a class="a"href="/newidea">
-    <button mat-button class="button">
-      <mat-icon>how_to_reg</mat-icon>
-      <b>Sign up</b>
-    </button>
-      </a>
+    <form [formGroup]="form">
+      <input type="text" name="email" formControlName="email" maxlength="77" placeholder="Email">
```
Comment (NeHerkus) [inline src/app/navigationbar/navigation-bar.component.html]: Convert values to consts

File: src/app/navigationbar/navigation-bar.component.ts [hunk 2]
```diff
@@ -1,16 +1,56 @@
...
+      this.login(this.form.value);
+    }
+    this.formSubmitAttempt = true;
+  }
+
+  login(user: User) {
+    if (!(user.name !== '' && user.password !== '')) {
+      return;
+    }
+    console.log('oba');
```
Comment (NeHerkus) [inline src/app/navigationbar/navigation-bar.component.ts]: No console logs

**Review by NeHerkus — CHANGES_REQUESTED** [review]
Some minor changes needed

**Review by NeHerkus — APPROVED** [review]
Everything fine

**Review by Pesimistas — APPROVED** [review]
All good

**Pull request merged**